app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <string name="app_name">DrawingApp</string>
-    <string name="new_layer_dialog_title">Create New Layer</string>
+    <string name="new_layer_dialog_title">Create / Edit Layer</string>
```

Comment:
Your title for your dialog should be smart enough to know which function it is doing, pass it in to your function based on context

Reply:
updated to handle